pages/common/git-filter-repo.md

```diff
@@ -0,0 +1,21 @@
+# git filter-repo
+
+> A versatile tool for rewriting Git history.
+> See also: `bfg`.
+> More information: <https://github.com/newren/git-filter-repo>.
+
+- Replace sensitive string in all files:
```

Comment:
nitpick maybe:
```suggestion
- Replace a sensitive string in all files:
```

Reply:
Since the replacement mechanism reads key-value pairs from the designated file, I would say that a plural form would be far more suitable, as it is even encouraged by design:

```suggestion
- Replace sensitive strings in all files:
```

Reply:
Would you please add an additional example describing the syntax for multiple key value pairs?

Reply:
Multiple key-value pairs have be inside a file, can you please point me to some example in `tldr` that describes contents of an input file needed to run a command?

This would enable to add other important `git-filter-repo` commands here, for instance ones that use `.mailcap` file to replace author/email in commit history.

Reply:
Hmmm, describing the format of the file if it's specific would belong in the documentation I think. So to this end it's perhaps best to stick to a single string here in this example instead of trying to expand to multiple strings?

Reply:
Thanks sbrl.

Unless if there is an easy way to specify multiple strings without specifying an input file, I would recommend keeping 
`- Replace sensitive string in all files:`